parser/parser.go

```diff
@@ -322,6 +324,19 @@ func (s *Parser) ParseLine(input string, cb ParsedTest) (test.Test, error) {
 	//
 	for arg, val := range result.Arguments {
 
+		// Is there a custom per-test override?
+		if arg == "maxRetries" {
```

Comment:
I appreciate you a) writing this and b) submitting it as a pull-request.

Now that I look more closely I think that `maxRetries` is perhaps not the best name.  Would you mind updating to rename it to just `retries`.

I think that matches the intent better, and also avoids the issue of mix-case, or having to use "max-retries", etc.

If you made that change I'd merge this, happily :)

Reply:
Done! :D